Events/SupervisorChanged.md

```diff
@@ -0,0 +1,61 @@
+# Supervisor Changed Event
+
+<table align="center">
+  <tr>
+    <td>Description</td>
+    <td>The HR system will raise an event for all subordinates of a person when they change, or for the person themself if it is known that their supervisor has changed.<br>Because employee job information is not public, a secur callback URL is provided for business applications that are authorized.</td>
+  </tr>
+  <tr>
+    <td>Examples</td>
+    <td>Sample for Supervisor Changed<br><pre>{
+  "events": {
+    "event": {
+      "event_header": {
+        "domain": "edu.byu",
+        "entity": "HR_Personal_Action",
+        "event_type": "Job Changed",
```

Comment:
Need to update this section from the Job Changed to Supervisor Changed event example.

Reply:
Whoops. Typing on autopilot apparently.